soroban-sdk/src/env.rs

```diff
@@ -207,9 +206,12 @@ impl Env {
     ///
     /// Return a [Result] instead of panicking.
     pub fn verify_sig_ed25519(&self, pk: &BytesN<32>, msg: &Bytes, sig: &BytesN<64>) {
-        internal::Env::verify_sig_ed25519(self, msg.to_object(), pk.to_object(), sig.to_object())
-            .try_into()
-            .unwrap()
+        let _ = internal::Env::verify_sig_ed25519(
+            self,
+            msg.to_object(),
+            pk.to_object(),
+            sig.to_object(),
+        );
```

Comment:
@jayz22 I'm pretty sure I don't need to check the return value of this host function as the host should stop the contract inside the host. Is that right?